docs/user-guide/src/OWNERS

```diff
@@ -28,7 +27,8 @@ emeritus_approvers:
  - hardys
  - russellb
  - Xenwar
+ - dukov
```

Comment:
you have to move this to emeritus_reviewers

Reply:
Done , thanks !